### PR TITLE
Optimize asset pipeline for CSS and Javascript

### PR DESCRIPTION
**Pull request by carsongee**
Create deterministic jasvascript loading
Minify all JS into single file with `build.py -p`
Minify all CSS into single file with `build.py -p`

### CODE REVIEW

**Comment by carsongee** [comment]
Closes https://github.com/starteam/starcellbio_html/issues/62 and should help with performance in production (about ~1.5 seconds faster when loaded locally vs. with individual files)

File: Vagrantfile [hunk 1]
```diff
@@ -26,7 +26,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # using a specific IP.
   config.vm.network "private_network", ip: "192.168.33.200"
   config.vm.network :forwarded_port, guest: 8100, host: 8100
-
+  config.vm.provider "virtualbox" do |vb|
+    vb.customize ["modifyvm", :id, "--memory", "1024"]
+  end
```
Comment (pdpinch) [inline Vagrantfile:31]: Is this increasing the memory on the VM?

**Comment by pdpinch** [comment]
@annagav @bdero please review

File: Vagrantfile (hunk 1 above)
Comment (carsongee) [inline Vagrantfile:31]: Yeah, It probably isn't strictly required, but I OOMed the box doing the minify with uglify-js

**Comment by bdero** [comment]
Thanks a lot for this. Tried it out and works in the vm out of the box. Waaaaaay faster with the -p flag. :money_with_wings:

**Pull request merged**